cmd/root.go

```diff
@@ -123,8 +123,8 @@ func initConfig() {
 		log.Printf("Using config file: %s\n\n", viper.ConfigFileUsed())
 	} else {
 		invalidConfigError := "Couldn't read config file"
-		fmt.Println("Error: ", invalidConfigError)
-		log.Fatal(invalidConfigError)
+		formattedError := fmt.Sprintf("%s: %s", invalidConfigError, err.Error())
+		log.Fatal(formattedError)
 		os.Exit(1)
```

Comment:
Not a big deal, but log.Fatal calls os.Exit(1) so you can lose this line.

Reply:
I'll just rebase on your PR :D